### ISSUE TITLE
Slack Integration

### ISSUE THREAD

**Issue by tainhenning** [tainhenning/levels-slack-scrape#4]
Will need to spike out how to connect our python code to slack, this will probably be blocked by AWS integration.

**Comment by Mohit-Doshi** [comment]
~Should we split this into subtasks? Test locally and then deploy on an instance?~

Never mind, just saw #5
